docs/guide/personalization/developer_guide/tracking_api.md

```diff
@@ -197,49 +291,65 @@ The format of the URL is: 
 
-### Basket Event
+### Basket event
 
-The basket event can be used to add products to a user's shopping cart. This event is especially useful if anonymized checkout is allowed or no recurring user identification is possible. By using the shopping cart products as input for getting recommendations, the empty profile or cold-start problem (meaning the user has no buy history) can be solved. The more valuable basket events instead of recent user clicks can be used to provide personalized recommendations. It also happens quite often that users "store" products on their shopping wishlist and plan to buy them later. With the help of this information, personalized shopping cart-based recommendations can be provided in the whole shop.
+The basket event can be used to add products to a user's shopping cart. 
+This event is especially useful if anonymized checkout is allowed or no recurring user 
+identification is possible. 
+By using the shopping cart products as input for getting recommendations, problems with an empty profile or 
+no buy history for the user can be solved. 
+The more valuable basket events instead of recent user clicks can be used to provide 
+personalized recommendations. 
+It also happens quite often that users "store" products on their shopping wishlist and plan 
+to buy them later. 
+With the help of this information, personalized shopping cart-based recommendations can be provided 
+in the whole shop.
 
 `GET https://event.yoochoose.net/api/[customerid]/basket/[userid]/[itemtypeid]/[itemid]`
 
-|Name|Description|Values|
-|---|---|---|
-|categorypath|The categorypath from where the item is placed into the shopping cart.|URL-encoded string|
-
-The categorypath parameter should be added for statistical reasons. Often articles or products are multihomed in a shop or a media/publisher site. In order to analyze from which location products were sold most often, the categorypath should be provided.
+There are no query string parameters for this event. 
 
-### Rate Event
+### Rate event
 
-Publishers, media or shops often allow commenting/rating products, articles or movies. If a user comments on an item, it indicates a special interest an this topic that has to be treated separately.
+Publishers, media or shops often allow commenting/rating products, articles or movies. 
+If a user comments on an item, it indicates a special interest in this topic that has 
+to be treated separately.
 
 The format of the URL is:
 
 `GET https://event.yoochoose.net/api/[customerid]/rate/[userid]/[itemtypeid]/[itemid]?rating=50`
 
-This can also be used for explicit ratings like a five star rating or any other liquor scale-based rating. The predefined rating can be submitted when the user comments on an item.
+This can also be used for explicit ratings like a five-star rating or any other liquor scale-based rating. 
```

Comment:
Liquor? That sound like a leftover from some example that vanished in the meantime.